lib/chef/knife/windows_helper.rb

```diff
@@ -27,7 +28,8 @@ class WindowsHelper < Knife
 
       banner "#{BootstrapWindowsWinrm.banner}\n" +
```

Comment:
I take that back, it does have the other cert commands since they have the knife windows pattern.